juntagrico/views_admin.py

```diff
@@ -47,6 +48,11 @@ def send_email_area(request):
     return send_email_intern(request)
 
 
+@any_permission_required('juntagrico.is_area_admin', 'juntagrico.can_send_mails')
```

Comment:
is_operationgroup also one to consider?

Reply:
No, `is_operationgroup` does not implicitly give access to send system emails

Reply:
originaly it would since the idea was that operation group allowed to see the complete admin menu, bust since i did not implement the any permission decorator  this idea got a bit watered down, i think the way to go is to remove the operationsgroup permission and add a a permission for each menu item, and the cooperative can decide how to group them. i think we have a tiket for that

Reply:
yes #199